packages/react-composites/tests/browser/callwithchat/app/index.tsx

```diff
@@ -71,18 +62,18 @@ function App(): JSX.Element {
     return <h3>ERROR: No endpoint set.</h3>;
   } else if (!threadId) {
     return <h3>ERROR: No threadId set.</h3>;
-  } else if (!callWithChatAdapter) {
+  } else if (!adapter) {
     return <h3>Initializing call and chat adapters...</h3>;
   }
 
   return (
     <>
-      {!callWithChatAdapter && 'Initializing call-with-chat adapter...'}
-      {callWithChatAdapter && (
+      {!adapter && 'Initializing call-with-chat adapter...'}
```

Comment:
Can remove this, should be covered in the `else if (!adapter) { return ...` above

Reply:
Being lazy as this is unrelated to current PR.